app/assets/javascripts/pretty-text/engines/discourse-markdown/code.js

```diff
@@ -77,10 +79,20 @@ function render(tokens, idx, options, env, slf, md) {
 
 export function setup(helper) {
   helper.registerOptions((opts, siteSettings) => {
-    opts.defaultCodeLang = siteSettings.default_code_lang;
-    opts.acceptableCodeClasses = (siteSettings.highlighted_languages || "")
+    const languages = (siteSettings.highlighted_languages || "")
       .split("|")
-      .filter(Boolean)
+      .filter(Boolean);
+    const languageAliases = [];
+
+    languages.forEach((lang) => {
+      if (HLJS_ALIASES[lang]) {
+        languageAliases.push(HLJS_ALIASES[lang]);
```

Comment:
Have you tried using `concat` instead of `push` so that you don't have to use `flat()` afterwards?

Reply:
I had not tried that, good idea, made it so!